WordPress/Classes/Services/PlanService.swift

```diff
@@ -8,7 +8,14 @@ open class PlanService: LocalCoreDataService {
     @objc public func getWpcomPlans(_ success: @escaping () -> Void,
                           failure: @escaping (Error?) -> Void) {
 
-        let remote = PlanServiceRemote(wordPressComRestApi: WordPressComRestApi())
+        let accountService = AccountService(managedObjectContext: managedObjectContext)
+        var remote: PlanServiceRemote
```

Comment:
This can still be a `let` :) Swift is smart enough to figure that you've assigned the value to a constant in every possible branch before it's used, so it's fine to use it here.

```suggestion
        let remote: PlanServiceRemote
```

Reply:
True... it just makes me twitch a little to do that 😆   I'll change it tho.